tdrs-backend/tdpservice/search_indexes/management/commands/seed_records.py

```diff
@@ -0,0 +1,135 @@
+"""Management command for seeding parsed records into the postgres db."""
```

Comment:
so I thought you are overriding this, am I correct or not? because I can't find this command [here](https://github.com/django-es/django-elasticsearch-dsl)